src/OAuth2/Storage/Pdo.php

```diff
@@ -513,6 +514,14 @@ public function getBuildSql($dbName = 'oauth2_server_php')
           subject             VARCHAR(80),
           public_key          VARCHAR(2000) NOT NULL
         );
+        
+        CREATE TABLE {$this->config['jti_table']} (
+          issuer              VARCHAR(80)   NOT NULL,
+          subject             VARCHAR(80),
+          audiance            VARCHAR(80),
```

Comment:
@gwillings nice catch of that 👃🐛  typo :) 👍 
 audi**e**nce vs audi**a**nce
`Uncaught PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'audience' in 'where clause' in [..]\vendor\bshaffer\oauth2-server-php\src\OAuth2\Storage\Pdo.php on line`

audiance            VARCHAR(80),